bootstrap/eks/api/v1beta1/conversion_test.go

```diff
@@ -1,46 +0,0 @@
-/*
```

Comment:
Why was this test deleted? 🤔

Reply:
It's restored, see https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3757#discussion_r990466679